config/action.d/smtp.py

```diff
@@ -155,10 +157,19 @@ def _sendMessage(self, subject, text):
 		msg['To'] = self.toaddr
 		msg['Date'] = formatdate()
 
-		smtp = smtplib.SMTP()
+		smtp = smtplib.SMTP(self.host)
 		try:
 			self._logSys.debug("Connected to SMTP '%s', response: %i: %s",
 				self.host, *smtp.connect(self.host))
+
+			if self.ssl: # pragma: no cover
+				tls_result = smtp.starttls()[0];
+				if tls_result != 220: # pragma: no cover
+					self._logSys.error(
+						"Failed to starttls() on '%s' for user '%s': %s",
+						self.host, self.user, tls_result)
```

Comment:
Since TLS takes place before login, user is probably irrelevant here, isn't it?

Reply:
Definitely a fair point.  I figure the extra information doesn't hurt.  Though, I suppose if someone tries to setup two SMTP connections to the same host, even with different users, both TLS attempts should fail so the user probably doesn't add anything useful.

Want me to remove it?

Reply:
I don't think even the logging is necessarily at all, because raising of error should be enough too.

Reply:
Done.